managed/models/service_helpers.go

```diff
@@ -405,3 +405,48 @@ func ValidateServiceType(serviceType ServiceType) error {
 		return errors.Wrapf(ErrInvalidServiceType, "unknown service type '%s'", string(serviceType))
 	}
 }
+
+// ChangeStandardLabelsParams contains parameters for changing standard labels for a service.
+type ChangeStandardLabelsParams struct {
+	ServiceID      string
+	Cluster        *string
+	Environment    *string
+	ReplicationSet *string
+	ExternalGroup  *string
+}
+
+// ChangeStandardLabels changes standard labels for a service.
+func ChangeStandardLabels(q *reform.Querier, id string, labels ServiceStandardLabelsParams) error {
+	s, err := FindServiceByID(q, id)
+	if err != nil {
+		return err
+	}
+
+	columns := []string{}
+
+	if labels.Cluster != nil {
+		columns = append(columns, "cluster")
```

Comment:
I'm still suggesting to use constants at the top of file like `const serviceClusterField = "cluster"` and cover all such parts with tests. Or at least just add tests. Otherwise if somebody changes string values and we miss it on review, we will never know about that until users report bug.

If you have any other ideas how to protect from bugs, please implement them.

Reply:
I understand the intention. This PR does not make the situation worse though.
There's numerous places where we already do the same, such as:

https://github.com/percona/pmm/blob/a9716c8232669deea5c6eb548982e4bc9fcaa36c/managed/models/node_helpers.go#L139
https://github.com/percona/pmm/blob/a9716c8232669deea5c6eb548982e4bc9fcaa36c/managed/models/node_helpers.go#L163

We are not going to improve the current state by applying a new approach to the column names here.
I don't expect we'll be able to change the column names in the future without a major refactoring of the whole codebase.

Either way, I added tests which shall cover changing any label and making sure the column names match.